packages/network-controller/src/NetworkController.ts

```diff
@@ -467,15 +476,12 @@ export type NetworkControllerOptions = {
 export const defaultState: NetworkState = {
   selectedNetworkClientId: NetworkType.mainnet,
   networkId: null,
-  networkStatus: NetworkStatus.Unknown,
   providerConfig: {
     type: NetworkType.mainnet,
     chainId: ChainId.mainnet,
     ticker: NetworksTicker.mainnet,
   },
-  networkDetails: {
-    EIPS: {},
-  },
+  networksMetadata: {},
```

Comment:
@Gudahtt @mcmire what do you guys think of this name we went with?

Reply:
I left a comment on this below.